go/test/endtoend/reparent/newfeaturetest/reparent_test.go

```diff
@@ -0,0 +1,237 @@
+/*
+Copyright 2020 The Vitess Authors.
+
```

Comment:
Not sure if this file should in a `incompatible` package instead of `newfeature`

Reply:
What I meant by `newfeature` was that these are the tests for the changes or new features that have gone into the reparent functions which will only work after all the components are upgraded. Basically, after release 13, all the tests in `newfeature` will move to the normal tests and new ones will go here.

Reply:
header should be 2022 😀

Reply:
Living in denial ft. me 😆

Reply:
I'll fix it